lazy_tensor_core/ts_native_functions.yaml

```diff
@@ -0,0 +1,34 @@
+backend: XLA
```

Comment:
This isn't gonna work out of the box- in addition to generating some boilerplate, the codegen also tries to register a bunch of stuff to the dispatcher, and it expects the passed-in backend to be a valid DispatchKey that it can register to.

We can talk more about what the codegen bits of the integration should look like at the sync. I probably need to look more at the LTC branch to see what it's doing, but it sounds like the codegen will be useful for generating most of the boilerplate, minus any dispatcher registrations.

Reply:
@bdhirsh I agree & was planning to discuss this topic during our meeting. We should de-hardcode names / namespaces and not force backends to lower all operations which have `_out` versions - currently we auto-generate those in a way which expects them to exist.